src/components/people/SelectedPersonResultCard.js

```diff
@@ -30,37 +27,25 @@ const DataGridWrapper = styled.div`
   padding: 30px;
 `;
 
-const PictureWrapper = styled.div`
+const UtilizerTagWrapper = styled.div`
+  align-items: flex-end;
   display: flex;
-  height: 242px;
-  position: relative;
-  width: 242px;
-`;
-
-const UserIconWrapper = styled.div`
-  align-items: center;
-  background-color: #e9ecef;
-  color: white;
-  display: flex;
-  flex: 1;
+  height: 200px; /* expected to be size of picture */
```

Comment:
In theory, with the previous comment, you could export the `getStyleVariation` to make those styles portable between multiple components without any extra legwork. If this `UtilizerTagWrapper` needed to change sizes with the picture in the future, you'd get that for free.